sycl/CMakeLists.txt

```
@@ -11,7 +11,7 @@ option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
 set(SYCL_MAJOR_VERSION 2)
 set(SYCL_MINOR_VERSION 0)
 set(SYCL_PATCH_VERSION 0)
-set(SYCL_DEV_ABI_VERSION 0)
+set(SYCL_DEV_ABI_VERSION 1)
```

```suggestion
set(SYCL_DEV_ABI_VERSION 0)
```

Adding new methods is not a breaking change. So, no need to uplift any versions.

---

Done.